test/unit/proxy_test.rb

```diff
@@ -73,6 +74,36 @@ def test_policy_chain
       assert_equal policy_chain, proxy.policy_chain
     end
 
+
+    def test_policy_chain_with_backend_apis
+      rolling_updates_on
+
+      account = FactoryBot.create(:simple_provider)
+      service = FactoryBot.create(:simple_service, account: account, act_as_product: true)
+      null_backend_api = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://foo.baz')
+      null_backend_api.update_columns(private_endpoint: '')
+      backend_api1 = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://private-1.example.com')
+      backend_api2 = FactoryBot.create(:backend_api, account: account, private_endpoint: 'https://private-2.example.com')
+      FactoryBot.create(:backend_api_config, path: '/null', backend_api: null_backend_api, service: service)
+      FactoryBot.create(:backend_api_config, path: '/foo', backend_api: backend_api1, service: service)
+      FactoryBot.create(:backend_api_config, path: '/bar', backend_api: backend_api2, service: service)
+      policy_chain =  [
+        {"name"=>"routing", "version"=>"builtin", "enabled"=>true,
```

Comment:
The "enabled" field is not used in APIcast. I guess it's something that only Porta uses?

Reply:
Yes true, but we have it since last year.
It is probably better in a next PR to remove it

Reply:
:+1: